parcellab/microservice/templates/poddisruptionbudget.yaml

```diff
@@ -1 +1,5 @@
-{{- include "common.poddisruptionbudget" . }}
+{{- if .Values.podDisruptionBudget.enabled -}}
```

Comment:
Isn't the previous version the same since the enabled check and full name is applied in the common template as well?

Reply:
yes you are right, rolled back